modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,320 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cronjob that starts a single backup each time it triggers.
+
+You can set the method of backup retention to `RetentionNumber`, `RetentionSize`, or `""`. If unspecified or set to `""`, the default retention type is `RetentionSize` with 15 maximum backups. Specifying a retention type overrides the default value.
```

Comment:
probably better to put this after (6) where you're actually configuring the retention?

we also have a default for RetentionSize, which is 10gigs. 

https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go#L207-L221